CartoDB/carto-react-lib
# Padawannn/workers

To test this PR:

- Use padawannn/worker-test branch in carto-react-template
- The current deck gl version crash when you use the binary mode and you get an empty tile so you should use the 8.4.6 deck.gl version
